idb/postgres/postgres_migrations.go

```diff
@@ -575,18 +637,48 @@ func initM5AccountData() *m5AccountData {
 	}
 }
 
+// accountTransactions pages through account transactions to avoid caching the entire result set. Unfortunately, this
+// also prevents us from benefiting from having a cache of the entire result set.
+func accountTransactions(ctx context.Context, db *IndexerDb, address types.Address, finalRound uint64) <-chan idb.TxnRow {
+	out := make(chan idb.TxnRow, 10)
+
+	go func() {
+		defer close(out)
+		token := ""
+		for true {
+			txnrows := db.Transactions(ctx, idb.TransactionFilter{
+				Address:   address[:],
+				MaxRound:  finalRound,
+				Limit:     1,
```

Comment:
I'd do a larger batch, ~1000, fetch them all into RAM quickly so as to let the sql transaction close and not be held open, then yield results from that buffer to the output chan

Reply:
oops, this low value was for testing.